hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/app.tsx

```diff
@@ -32,22 +32,23 @@ import classNames from 'classnames';
 import Loader from '@/v2/components/loader/loader';
 
 import './app.less';
+import NotFound from '@/v2/pages/notFound/notFound';
```

Comment:
So this is new page for `notFound 404`, but if it is imported here, how it is being used across Recon for all APIs in generic way ?